components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/util/IdentityTenantUtil.java

```diff
@@ -320,4 +320,26 @@ public static int getTenantIdOfUser(String username) throws IdentityRuntimeExcep
             return tenantId;
         }
     }
+
+    /**
+     * Get the tenant name from the thread local properties.
+     *
+     * @return Tenant name from the context.
+     */
+    public static String getTenantDomainFromContext() {
+
+        String tenantDomain = (String) IdentityUtil.threadLocalProperties.get().get
+                (IdentityCoreConstants.TENANT_NAME_FROM_CONTEXT);
+        return tenantDomain;
+    }
+
+    /**
+     * Checks whether the tenant URL support is enabled.
+     *
+     * @return true if the config is set to true, false otherwise.
+     */
+    public static boolean enableTenantQualifiedUrls() {
```

Comment:
isTenantQualifiedUrlsEnabled
Otherwise the method defines an action but it should denote it's a check